bin/reth/src/commands/node/mod.rs

```diff
@@ -290,42 +244,49 @@ mod tests {
 
     #[test]
     fn parse_discovery_port() {
-        let cmd = NodeCommand::<()>::try_parse_from(["reth", "--discovery.port", "300"]).unwrap();
+        let cmd =
+            NodeCommand::<NoArgs>::try_parse_from(["reth", "--discovery.port", "300"]).unwrap();
         assert_eq!(cmd.network.discovery.port, 300);
     }
 
     #[test]
     fn parse_port() {
-        let cmd =
-            NodeCommand::<()>::try_parse_from(["reth", "--discovery.port", "300", "--port", "99"])
-                .unwrap();
+        let cmd = NodeCommand::<NoArgs>::try_parse_from([
+            "reth",
+            "--discovery.port",
+            "300",
+            "--port",
+            "99",
+        ])
+        .unwrap();
         assert_eq!(cmd.network.discovery.port, 300);
         assert_eq!(cmd.network.port, 99);
     }
 
     #[test]
     fn parse_metrics_port() {
-        let cmd = NodeCommand::<()>::try_parse_from(["reth", "--metrics", "9001"]).unwrap();
+        let cmd = NodeCommand::<NoArgs>::try_parse_from(["reth", "--metrics", "9001"]).unwrap();
```

Comment:
The NoArgs explicit arg is not needed given above default config right

Reply:
for some reason it still is